src/app/app.component.ts

```diff
@@ -6,11 +6,19 @@ import { environment } from "../environments/environment";
   selector: "app-root",
   templateUrl: "./app.component.html",
 })
-export class AppComponent implements AfterViewChecked {
+export class AppComponent implements OnInit, AfterViewChecked {
+  private lang: string;
+
   constructor(private translate: TranslateService) {
     this.translate.addLangs(environment.languages);
-    const lang = localStorage.getItem("lang") || environment.languages[0];
-    this.translate.use(lang);
+    if (localStorage.getItem('lang')) {
+      this.lang = localStorage.getItem('lang')
+    } else if (navigator.language.substring(0, 2) === 'en' || navigator.language.substring(0, 2) === 'ru') {
```

Comment:
Everything is fine, but this part should have been a bit complex (see https://github.com/open-genes/open-genes-frontend/issues/5 description)

Reply:
I'll merge it, but I leave a task open. Maybe it's too much overhead.